week-09-js/var.js

```diff
@@ -0,0 +1,78 @@
+console.log("Hi!")
+
+
+getUsers();
+
+function renderListItem(userData) {
+  const li = document.createElement("li");
+  li.innerText = `${userData.lname} ${userData.fname} ${userData.Mphone} ${userData.street} ${userData.number} ${userData.City} ${userData.Country}`;
+  const ol = document.getElementById("fname","lname","Mphone","street","number","City","Country");
+  ol.appendChild(li);
+
+  const deleteElement = document.createElement("span");
+  deleteElement.innerText = " x ";
+  li.appendChild(deleteElement);
+
+  deleteElement.addEventListener("click", function() {
+    deleteUser(userData.id);
+  })
+}
+
+function renderUsers(data) {
+  
+  for (const user of data) {
+    renderListItem(user);
+  }
+}
+
+
+function processResponse(response) {
+  return response.json();
+}
+
+
+function getUsers() {
+  fetch("https://contact-agenda-rest-api.herokuapp.com/users")
+  .then(processResponse)
+  .then(renderUsers);
+}
+
+
+function getNewUserData(data){
+  
+  renderListItem(data);
+}
+
+
+function sendNewUser() {
+  const newUser = {
```

Comment:
the structure of the 'newUser' should have a address key in which there should be another object that have the keys like street, number, city, etc